packages/flutter/lib/src/cupertino/sheet.dart

```diff
@@ -135,6 +135,7 @@ Future<T?> showCupertinoSheet<T>({
   required BuildContext context,
   required WidgetBuilder pageBuilder,
   bool useNestedNavigation = false,
+  bool enableDrag = true,
```

Comment:
Could you please add something to the documentation of this function explaining `enableDrag`? Probably in the same paragraph as: "The whole sheet can be popped at once by either dragging down on the sheet, or calling [CupertinoSheetRoute.popSheet]." or that section should be modified.

Reply:
fix this commit

https://github.com/flutter/flutter/pull/163923/commits/5c9af746636e8497ebe7ca5576eb4fb6849a2a8a